packages/vue/src/composables/useSessionList.ts

```diff
@@ -35,14 +35,14 @@ export const useSessionList: UseSessionList = () => {
   const { clerk, clientCtx } = useClerkContext();
 
   const result = computed<UseSessionListReturn>(() => {
-    if (!clientCtx.value) {
+    if (!clientCtx.value || !clerk.value) {
```

Comment:
Okay, the issue is that we have the `clientCtx` value i[nitially set to an empty object](https://github.com/clerk/javascript/blob/3c225d90227141dc62d955e76c7f8e0202524bc7/packages/vue/src/plugin.ts#L39) that's why it passes the condition even if `clerk` is null.

Would you be able to update the [initial value to undefined](https://github.com/clerk/javascript/blob/3c225d90227141dc62d955e76c7f8e0202524bc7/packages/vue/src/plugin.ts#L39) and assert the type for now? (`client: undefined as unknown as ClientResource`)

This should be a safe update as we check the value of `clerk` in all of our composables except `useSessionList()`.

Thanks!